src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -722,7 +722,6 @@ zocl_xclbin_read_axlf(struct drm_zocl_dev *zdev, struct drm_zocl_axlf *axlf_obj,
 		vfree(axlf);
 		return -EFAULT;
 	}
-
 	write_lock(&zdev->attr_rwlock);
```

Comment:
Could you add a comment in before this lock to emphasize that "after this lock to unlock is an atomic context"?

Reply:
Added a comments